org.eclipse.wb.swt/src/org/eclipse/wb/internal/swt/model/property/editor/color/ColorPropertyEditor.java

```diff
@@ -453,11 +451,10 @@ public RegistryColorsPage(Composite parent,
           }
         }
         // create grid composite
-        ColorsGridComposite colorsGrid =
-            createColorsGroup(
-                this,
-                registryInfo.getVariableSupport().getTitle(),
-                infos.toArray(new ColorInfo[infos.size()]));
+        ColorsGridComposite colorsGrid = createColorsGroup(

```

Comment:
Format changes and real changes should not be mixed. please provide separate PR for this.

Reply:
Lars, maybe we can allow this for now and ask Marcel to take this into account for later pushes?